File: Oishi/Basics of Programming/FunctionsAndArrays/FindElement.cpp
#include<iostream>
using namespace std;


int main(){
    //write your code here
    int n;
    cin >> n;
    
    int arr[n];
    for (int i=0; i<n; i++) {
        int x;
        cin >> x;
        arr[i] = x;
    }
    
    int d;
    cin >> d;
    
    int i;
    for (i=0; i<n; i++) {
        if (arr[i] == d) {
            cout << i << endl;
            break;
        }
    }
    
    if (i == n) 
        cout << "-1" << endl;
}

File: V.Vennela/Basics of Programming/Functions and Arrays/FindElement.java
import java.io.*;
import java.util.*;

public class FindElement{

public static void main(String[] args) throws Exception {
    // write your code here
    Scanner sc = new Scanner(System.in);
    int n = sc.nextInt();
    int a[] = new int[n];
    for(int i=0;i<n;i++)
    {
        a[i]=sc.nextInt();
    }
    int key = sc.nextInt();
    int flag=1;
    int i=0;
    for(i=0;i<n;i++)
    {
        if(key==a[i])
        {
            flag=0;
            break;
        }
           
    }
    if(flag==1)
        System.out.println("-1");
    else
        System.out.println(i);
 }

}
